packages/ui-library/src/components/token-picker-button/index.tsx

```diff
@@ -36,30 +50,28 @@ const StyledKeyboardArrowDownIcon = styled(KeyboardArrowDownIcon)`
 `;
 
 interface TokenPickerButtonProps {
-  token: TokenWithIcon;
-  isSelected: boolean;
+  token?: TokenWithIcon;
   isLoading?: boolean;
-  onClick: ButtonProps['onClick'];
+  onClick?: ButtonProps['onClick'];
   showAction: boolean;
   defaultText?: string;
   disabled?: boolean;
 }
 
 const TokenPickerButton = ({
   token,
-  isSelected,
   isLoading,
   onClick,
   showAction = true,
   defaultText,
   disabled,
 }: TokenPickerButtonProps) => (
   <StyledTokenPickerButton disabled={disabled} onClick={onClick}>
-    {token.icon}
-    <StyledActiveLabel $isSelected={isSelected}>
+    {token?.icon || <StyledEmptyTokenIcon $realSize={SPACING(7)} />}
```

Comment:
I dont find the emptyTokenIcon size flexible enough. We could do one of two:
1. Always expect a token. If the consumer of this component doesn't have one, it should do 
`token = {...toToken({icon: <TokenIcon size={X} />})}`. The empty token in TokenIcon can be this StyledEmptyTokenIcon
2. Get the realSize as a param
What do you think?